### ISSUE TITLE
mantra-js.yaml creates Typescript files

### ISSUE THREAD

**Issue by tomfxs** [mantrajs/mantrajs-atom-package#25]

**Comment by tomitrescak** [comment]
Can you tell me which files? Can you revise follwoing template file and let me know? 

https://github.com/mantrajs/mantrajs-atom-package/blob/master/configs/mantra-js.yaml

**Comment by tomfxs** [comment]
All index-files e.g. actions/index.ts have .ts extensions, shouldn't they have .js?

**Comment by tomitrescak** [comment]
Absolutely, just pushed a new version. LEt me know if there is anything else. Thanks for the feedback!

**Comment by tomfxs** [comment]
Thanks, you are doing a great job!

**Comment by tomfxs** [comment]
Im the previous version there were main.js in the client and server root, now created as index.js
But now problem, i will rename them in my existing projects. routes had the extension .jsx, didn't they?
